content/automate/ManageIQ/Infrastructure/VM/Transform/StateMachines/VmImportWaitForVm.class/__methods__/wait_for_imported_vm.rb

```diff
@@ -1,36 +1,42 @@
+#
+# Description: This method checks if the VM has been successfully imported
+#
+
 module ManageIQ
   module Automate
-    module System
-      module Event
-        module EmsEvent
-          module RHEVM
-            class UpdateVmImportStatus
```

Comment:
@pkliczewski UpdateVmImportStatus cannot work as it is because when IMPORTEXPORT_IMPORT_VM event arrives nobody guarantees that the VM object is accessible. That's why I'm replacing the UpdateVmImportStatus method with a state machine that will wait until the VM object is accessible and only after that will update its import status.